.github/workflows/cicd.yaml

```diff
@@ -94,3 +94,32 @@ jobs:
       build_image: ${{ needs.build.outputs.build_image }}
       skip_integration_tests: ${{ needs.conditionals.outputs.skip_integration_tests }}
       cosign_public_key: ${{ needs.build.outputs.cosign_public_key }}
+
+  check-commit-message:
```

Comment:
Can you make that a step in `.reusable_compliance`?

Reply:
Conditionals would be the other option

Reply:
Wouldn't put it in conditionals as that would then fail build and would necessitate us making conformant commits even during testing, as opposed to at the end